Bert Johnson (johnb111) is listed in batting slots 1 and 4 and as the starter
in both left field and right field. In every other 1934 Newark Dodgers box
score, including game 1 of this doubleheader, Buddy Burbage (burbb102) leads
off in left field and Johnson bats fourth in right field. Burbage does not
otherwise appear in this game. The slot 1 start record, bline, and the left
field dline are reassigned to Burbage; Johnson keeps slot 4 in right field.

--- a/ngl_b/1934.EBR
+++ b/ngl_b/1934.EBR
@@ -31,6 +31,6 @@
 start,day-l101,"Leon Day",0,9,1
-start,johnb111,"Bert Johnson",1,1,79
+start,burbb102,"Buddy Burbage",1,1,7
 start,arnop101,"Paul Arnold",1,2,8
 start,dandr101,"Ray Dandridge",1,3,5
-start,johnb111,"Bert Johnson",1,4,79
+start,johnb111,"Bert Johnson",1,4,9
 start,starj102,"Jim Starks",1,5,3
@@ -52,3 +52,3 @@
 stat,bline,mitcb103,0,9,2,1,1,1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
-stat,bline,johnb111,1,1,1,5,2,2,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
+stat,bline,burbb102,1,1,1,5,2,2,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
 stat,bline,johnb111,1,4,1,5,3,3,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1,-1
@@ -82,3 +82,3 @@
 stat,dline,mitcb103,0,1,1,-1,-1,-1,0,-1,-1,0
-stat,dline,johnb111,1,1,7,21,-1,-1,0,-1,-1,0
+stat,dline,burbb102,1,1,7,21,-1,-1,0,-1,-1,0
 stat,dline,johnb111,1,1,9,21,-1,-1,0,-1,-1,0
